spec/dummy/db/schema.rb

```diff
@@ -11,13 +11,13 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20150727145934) do
+ActiveRecord::Schema.define(version: 20150727151245) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
   create_table "event_capture_events", force: :cascade do |t|
-    t.datetime "emitted_at"
+    t.datetime "emitted_at",  null: false
```

Comment:
Great question.  It seems that in all of the following applications on **staging**:
*think_feel_do, think_feel_do_so, and sunnyside*; along with all the following on **production**: *think_feel_do, think_feel_do_so, sunnyside, and stepped_care_reboot*, there are no captured events where `emitted_at` or a `recorded_at` are null.